android/app/src/main/java/com/podverse/PVUnifiedPushModule.java

```diff
@@ -132,7 +142,12 @@ public void onReceive(Context context, Intent intent) {
                 emitReactEvent(getReactApplicationContext(), UPMessage);
             }
         };
-        getReactApplicationContext().registerReceiver(receiver, intentFilter);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            getReactApplicationContext().registerReceiver(receiver, intentFilter, Context.RECEIVER_NOT_EXPORTED);
+        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            getReactApplicationContext().registerReceiver(receiver, intentFilter);
+        }
```

Comment:
Could you explain this briefly?

Reply:
I was getting an IDE warning regarding not setting `RECEIVER_NOT_EXPORTED` but it appears to need Android 13 and will be a requirement for Android 14 from what I read.  Actually not sure about Android 8 support anymore because i can't find the warning so I'll revert that part